### PR TITLE
Add meeting minutes for January 2023

### PR DESCRIPTION
**Pull request by tsyesika**

### CODE REVIEW

**Review by zenhack — APPROVED** [review]

**Review by ocdtrekkie — APPROVED** [review]
I was able to follow the scribing during the meeting and viewed the recording later. Scribing was excellent for this meeting.

**Comment by ocdtrekkie** [comment]
Did we have a decision on out-of-band minute approval? I feel like if we can get a quorum of approvals on here from multiple organizations participating (Spritely, Agoric, Sandstorm), it should be safe to merge these quicker and not to waste time on this during the actual meetings. Presumably if we fail to meet an adequate approval standard here, we can address any concerns with the meeting notes at the next meeting.

File: meeting-minutes/2023-01-31.md [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+
+isd: +1
+
+alan: +1
+
+**APPROVED**: meeting minutes for 2022-12-06
+
+Jessica: meeting minutes approved.
+
+Jessica: we're waiting on Mark Miller but Kris Kowel from Agoric is now here
```
Comment (dckc) [inline meeting-minutes/2023-01-31.md:56]:
```suggestion
Jessica: we're waiting on Mark Miller but Kris Kowal from Agoric is now here
```

File: meeting-minutes/2023-01-31.md [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+  - Christine Lemmer Webber (cwebber)
+  - Dale Schumacher
+  - David Thompson (dthompson)
+  - Edward Platt (edplatt)
+  - Ian Denhardt (isd)
+  - Jessica Tallon (Jessica)
+  - Jonathan Rees (jar286)
+  - Kris Kowal (kriskowal)
+  - Mark Miller (markm)
+  - Mike Stay (Mikestay)
```
Comment (dckc) [inline meeting-minutes/2023-01-31.md:17]:
```suggestion
  - Mike Stay (pyrocto)
```

**Review by dckc — APPROVED** [review]
neither of my suggestions is critical

**Comment by dckc** [comment]
> Did we have a decision on out-of-band minute approval?

I see 2 decisions in the meeting record; neither regards out-of-bands approval of minutes.

I still think it's a reasonable thing to do. It's not like this group has a formal decision-making process that requires approved minutes as opposed to shared notes.

**Comment by tsyesika** [comment]
I've fixed the two typos that @dckc has suggested, thank you for reviewing the minutes!

I agree that we should probably do it out of band. I propose that we vote on these minutes in the meeting and we also take a vote on moving this to out of band, so that future minutes can be approved by votes on the github PR.

**Comment by tsyesika** [comment]
These were approved in the Feb 2023 meeting!

**Pull request merged**